src/components/slide/slide.tsx

```diff
@@ -154,7 +150,6 @@ const Slide = (props: SlideProps): JSX.Element => {
     return result;
   }, [slideTransition, transition]);
 
-  const inOverviewMode = Object.entries(frameOverrideStyle).length > 0;
```

Comment:
`frameOverrideStyle` has entires in both overview and print / export modes, so `isOverviewMode` was `isInOverviewOrPrintMode`. To distinguish between overview and print modes I've added booleans for both to `DeckContext` which I'm now using instead.